### DISCUSSION TITLE
Some enhancements

### DISCUSSION THREAD

**Discussion by bruste77** [JayQueue/MyWhoosh2Garmin#3] [Ideas]
Hi,
thanks for your great script!
What I'd like to have in future is:
- Same activity title as MyWhoosh and Strava
- Correct date time 
- Option to remove the equipment (garmin connect doesn't have a specific equipment for virtual cycling)

**Comment by JayQueue** [comment]
hi,
- I'm looking into the activity name but it's not that straightforward since you just upload the .fit file. There is no "activity name" in that file.
- And what's the issue with the date/time? My activities have the correct date time.
- Going to dive in the .fit to see if there's equipment

-J

**Comment by bruste77** [comment]
A
![ActivityOnGarminConnect](https://github.com/user-attachments/assets/479eeb32-c07e-47aa-bdb7-9d6811640dc4)
![ActivityOnStrava](https://github.com/user-attachments/assets/bf0aeff0-7489-4fa8-a738-4315860a1b1e)

The attached images are dates and times of an activity done yesterday evening (Garmin connect and Strava). Date and time taken from Strava are correct. Garmin connect shows 4 in the morning. I've checked options about locale on my whoosh, but I did find anything...I'll try on Garmin connect.

About equipment I thinks it's added by Garmin, so maybe you cannot do anything
